crates/apub/src/activities/deletion/mod.rs

```diff
@@ -245,6 +271,7 @@ async fn receive_delete_action(
   object: &Url,
   actor: &ObjectId<ApubPerson>,
   deleted: bool,
+  person_remove_data: Option<bool>,
```

Comment:
This should be changed to `delete_content` , or `purge_content`.

remove_data is a term we use for admins removing a users data, which only sets the removed fields to true.

Reply:
Its a bit confusing then because we are federating it as `remove_data` now. I renamed it to `do_purge_user_account`, like the function it calls.